apps/neuvector/stg/01/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ../base
+
+# patchesStrategicMerge:
+#   - ../../neuvector/stg/01.yaml
```

Comment:
why is this commented out?

Reply:
ithc has it. When I tested locally, that file commented out is still included without the patch and it doesn't work with the patch.